runtime/hydradx/src/system.rs

```diff
@@ -471,12 +473,11 @@ impl pallet_transaction_payment::Config for Runtime {
 
 impl pallet_transaction_multi_payment::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type AcceptedCurrencyOrigin = SuperMajorityTechCommittee;
-	type Currencies = Currencies;
-	type RouteProvider = Router;
-	type OraclePriceProvider = OraclePriceProvider<AssetId, EmaOracle, LRNA>;
+	type AssetId = AssetId;
+	type Balance = Balance;
+	type AuthorityOrigin = SuperMajorityTechCommittee;
+	type PriceProvider = OmnipoolSpotPriceProvider<Runtime>;
```

Comment:
with this we support only assets that are in omnipool as payment assets, we should support every asset we have price for in oracle and aditionally even new assets that dont have any price in oracle yet (till shitcoin casino)